analyzers/src/SonarAnalyzer.CSharp/Rules/UninvokedEventDeclaration.cs

```diff
@@ -73,8 +73,7 @@ private void RaiseOnUninvokedEventDeclaration(SymbolAnalysisContext context)
             var possiblyCopiedSymbols = GetPossiblyCopiedSymbols(removableDeclarationCollector);
 
             removableEventFields
-                .Where(x => !invokedSymbols.Contains(x.Symbol))
-                .Where(x => !possiblyCopiedSymbols.Contains(x.Symbol))
+                .Where(x => !invokedSymbols.Contains(x.Symbol) && !possiblyCopiedSymbols.Contains(x.Symbol))
```

Comment:
_While seeing it like this:_ Do we need to different sets to do the same thing twice? Let imagine
* Variable `usedSymbols` instead
* `GetInvokedEventSymbols` to return `IEnumerable` 
* `GetPossiblyCopiedSymbols` to return `IEnumerable` 
* `usedSymbols = GetInvokedEventSymbols(..).Concat(GetPossiblyCopiedSymbols(..)).ToHashSet()`